src/main/resources/changelog/scripts/v-1.0.0/resource_graph/functions/check_unique_active_srs_id.sql

```diff
@@ -0,0 +1,31 @@
+--liquibase formatted sql
+
+--changeset pavel_bobylev:check_unique_active_srs_id dbms:postgresql splitStatements:false
+do $do$
+BEGIN
```

Comment:
Instead of this trigger, is it better to ensure that srs_id is unique? 

(We will ensure that only the latest resource will have an entry in folio_metadata table. We will remove the entry in folio_metadata for the old resouce)

Reply:
yep, and I think this is part of your comment #1, so I will handle this too